### PR TITLE
Combination

### PR DESCRIPTION
**Pull request by lewismcnish**
## Overview

(**Why** you made the changes and **What** they achieved)

## Changes

- ...
- ...
- ...

## Testing

(**How** you tested it and **proof** it works)

## Additional Information

(**Extra Notes** / **Known Issues** / **Discussion Points** / **Request Feedback**)

### CODE REVIEW

File: cloud-mask/cloud-mask.ipynb [hunk 1]
```diff
@@ -62,20 +62,15 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3.11.0 64-bit",
+   "display_name": "Python 3",
```
Comment (CameronFraserr) [inline cloud-mask/cloud-mask.ipynb:65]: Why Python?

File: cloud-mask/cloud-mask.ipynb [hunk 2]
```diff
@@ -62,20 +62,15 @@
...
-   "version": "3.11.0"
+   "version": "3.10.11"
   },
-  "orig_nbformat": 4,
-  "vscode": {
-   "interpreter": {
-    "hash": "aee8b7b246df8f9039afb4144a1f6fd8d2ca17a180786b69acc140d282b71a49"
-   }
-  }
+  "orig_nbformat": 4
```
Comment (CameronFraserr) [inline cloud-mask/cloud-mask.ipynb:73]: Why 4? Is this a magic number?

**Review by CameronFraserr — CHANGES_REQUESTED** [review]
Overall looks good, just a few minor tweaks

**Comment by JazzyMaxine** [comment]
This PR was created as a demonstration IIUC. I'll close it but if I'm mistaken feel free to reopen.

**Pull request closed without merging**